designer/client/component-create.js

```diff
@@ -6,6 +6,13 @@ import ComponentTypes from 'digital-form-builder-engine/src/component-types'
 class ComponentCreate extends React.Component {
   state = {}
 
+  componentDidMount () {
+    const { data, onInit } = this.props
+    data.getId().then(id => {
+      this.setState({ id }, () => { onInit && onInit() })
```

Comment:
minor, onInit should be named something like just init() or initComponent() (or whatever it's trying it init). currently looks like it's an event handler

Reply:
onInit has now gone away. Good Riddance!